contribulyze/contribulyze.sh

```diff
@@ -2,27 +2,72 @@
 #
 # Generate contribulyse.py reports, for all git repositories, on a number of different time periods.
 #
-# Example run in docker
+# Example run in docker (from the cassandra-builds/contribulyze directory)
+#
 # docker run -t -v`pwd`/build/html:/tmp/contribulyze-html -v`pwd`:/contribulyze apache/cassandra-testing-ubuntu2004-java11-w-dependencies bash -lc 'pip3 install --quiet python-dateutil ; cd /contribulyze ; bash contribulyze.sh '
 #
 
 script_dir="$( cd "$( dirname "${BASH_SOURCE[0]}" )" &> /dev/null && pwd )"
 mkdir -p /tmp/contribulyze-repos
 mkdir -p /tmp/contribulyze-html
 cd /tmp/contribulyze-repos
-for repo in "https://github.com/apache/cassandra.git" "https://github.com/apache/cassandra-dtest.git" "https://github.com/apache/cassandra-builds.git" "https://github.com/apache/cassandra-website.git" ; do
+
+repos=("https://github.com/apache/cassandra.git" "https://github.com/apache/cassandra-dtest.git" "https://github.com/apache/cassandra-builds.git" "https://github.com/apache/cassandra-in-jvm-dtest-api.git" "https://github.com/apache/cassandra-harry.git" "https://github.com/apache/cassandra-website.git" "https://github.com/datastax/java-driver.git" "https://github.com/datastax/python-driver.git")
+
+groups=('..' 'website_and_docs' 'build_and_tools' 'packaging_and_release' 'test_and_ci' 'cassandra_src' 'java-driver' 'python-driver')
```

Comment:
nit: as far as I understand you use ".." as a kind of default/root group that keeps the previous behavior and paths. It slightly confused me initially, so I feel a small comment might be helpful here.

Reply:
done.